Octokit.Tests.Integration/Clients/RepositoryContentsClientTests.cs

```diff
@@ -128,5 +128,53 @@ public class TheGetContentsMethod
                 Helper.DeleteRepo(repository);
             }
         }
+
+        [IntegrationTest]
+        public async Task GetsArchiveLinkAsTarball()
+        {
+            var github = new GitHubClient(new ProductHeaderValue("OctokitTests"))
```

Comment:
:lipstick: just use `Helper.GetAuthenticatedClient()` for these integration tests